docs/resources/smn_subscription.md

```diff
@@ -34,16 +34,21 @@ resource "huaweicloud_smn_subscription" "subscription_2" {
 The following arguments are supported:
 
 * `region` - (Optional, String, ForceNew) The region in which to create the SMN subscription resource. If omitted, the
-  provider-level region will be used. Changing this creates a new SMN subscription resource.
+  provider-level region will be used. Changing this parameter will create a new resource.
 
-* `topic_urn` - (Required, String, ForceNew) Resource identifier of a topic, which is unique.
+* `topic_urn` - (Required, String, ForceNew) Specifies the resource identifier of a topic, which is unique.
+  Changing this parameter will create a new resource.
 
-* `endpoint` - (Required, String, ForceNew) Message endpoint. For an HTTP subscription, the endpoint starts with http:
-  //. For an HTTPS subscription, the endpoint starts with https://. For an email subscription, the endpoint is a mail
-  address. For an SMS message subscription, the endpoint is a phone number.
+* `protocol` - (Required, String, ForceNew) Specifies the protocol of the message endpoint. Currently, email, sms, http,
+  https, callnotify and functionstage are supported. Changing this parameter will create a new resource.
 
-* `protocol` - (Required, String, ForceNew) Protocol of the message endpoint. Currently, email, sms, http, and https are
-  supported.
+* `endpoint` - (Required, String, ForceNew) Message endpoint. Changing this parameter will create a new resource.
+  + **For an HTTP subscription**: the endpoint starts with http://.
+  + **For an HTTPS subscription**: the endpoint starts with https://.
+  + **For an email subscription**: the endpoint is an mail address.
+  + **For an SMS message subscription**: the endpoint is a phone number.
+  + **For a callnotify subscription**: the endpoint is a phone number.
+  + **For a functionstage subscription**: the endpoint is a function.
```

Comment:
function and workflow of functionstage is different

Reply:
fixed